downstream/modules/platform/proc-controller-getting-started-inventory.adoc

```diff
@@ -0,0 +1,21 @@
+[id="controller-getting-started-inventory"]
+
+= Creating a new Inventory
+
+The Inventories window displays a list of the inventories that are currently available. 
+You can sort the inventory list by name and searched type, organization, description, owners and modifiers of the inventory, or additional criteria.
+
+.Procedure
+. To view existing inventories, click btn:[Inventories] in the navigation bar.
```

Comment:
I'm not sure a) whether they're buttons to click, or menu items to select. Or whether it's a navigation bar or the navigation menu. Perhaps we could check with Donna on the accessibility implications?